### DISCUSSION TITLE
what is login URI in lobe chat

### DISCUSSION THREAD

**Discussion by snowden-fu** [lobehub/lobehub#2546] [General Question] [answered]
if my domain is `example.com`, is login URI `example.com/login`? or `/authorize`?
<img width="436" alt="image" src="https://github.com/lobehub/lobe-chat/assets/24203166/6f411ddb-bffd-4889-a077-55fc61107c60">

**Comment by arvinxx** [comment]
@cy948 please help~

**Comment by cy948** [comment, marked as answer]
Sorry, just set it `https://example.com/chat` ，it only use when user occur some errors.

**Reply by snowden-fu** [reply]
thx, looks good now. can you have a look at one more question? the question above is just from this: 
https://github.com/lobehub/lobe-chat/discussions/2551
